Packs/PrismaCloud/Layouts/layout-detailsV2-Azure_Storage_Misconfiguration.json

```
@@ -0,0 +1,646 @@
+{
+  "fromVersion": "4.1.0",
```

This should fix the validation failure
```suggestion
  "fromVersion": "5.0.0",
```